packages/cli/package.json

```diff
@@ -161,7 +161,7 @@
     "@types/minimatch": "5.1.2",
     "@types/mock-fs": "4.13.1",
     "@types/node": "18.17.17",
-    "@types/npm-packlist": "7.0.0",
+    "@types/npm-packlist": "3.0.0",
```

Comment:
Any reason for such a severe downgrade?

Reply:
Yes, there were some compilation problems, with some methods having changed signatures.